simple.ml

```diff
@@ -0,0 +1,26 @@
+let tbl = Hashtbl.create 1000000 in
+
+let process line =
+  line
+  |> String.lowercase_ascii
+  |> String.split_on_char ' '
+  |> List.iter (fun word ->
+      if (String.length word != 0) then
```

Comment:
You could also use `word != ""` here

Reply:
I actually used `word != ""` at first, but it was letting an empty word of some sort slip through, and piping to `python3 normalized.py` (via test.sh) was throwing an exception because of it. I'm still not quite sure why it was slipping through.

Reply:
Oh, that's interesting, I will look into it.